src/views/v2/components/fields/text.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * This template renders the Text or Textarea field.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/components/fields/text.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Text
+ */
+
+$field_name = tribe_tickets_ar_field_name( $ticket->ID, $field->slug );
+$field_id   = tribe_tickets_ar_field_id( $ticket->ID, $field->slug );
+$required   = tribe_tickets_ar_field_is_required( $field );
+$field      = (array) $field;
+$multiline  = isset( $field['extra'] ) && isset( $field['extra']['multiline'] ) ? $field['extra']['multiline'] : '';
+$disabled   = false;
+
+$classes    = [
+	'tribe-common-b1',
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 4 spaces

<sub>hash: b7eb6ffe84dab32addb5be8bed49fc26</sub>